model/storage_settings.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package model
+
+import (
+	"encoding/json"
+	"io"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/pkg/errors"
+)
+
+type StorageSettings struct {
+	Region string
+	Bucket string
+	Uri    string
+	Key    string
+	Secret string
+	Token  string
```

Comment:
Here you miss the annotations, e.g. `json:"region" bson:"region"`
This is the reason why you need the capitalized keys in the acceptance tests (`Region` vs. `region`).
Can you please apply the annotations here, and use lower case keys in the acceptance tests?

Reply:
Got it. Thanks!